assets/scss/extensions.scss

```diff
@@ -493,6 +493,13 @@ $img-height-mobile: 106px;
   font-size: var(--font-size-6);
   margin-bottom: 0;
 }
+.image-with-caption .p-button:enabled:hover {
```

Comment:
// add a comment to what this is doing. 
.p-image .p-button:enabled:hover { ...

the enlarge button is specific to the p-image component.

Reply:
ok done - but this was already under the image with caption section (there was already a comment above it)